qa/L0_client_memory_growth/test.sh

```diff
@@ -56,8 +56,8 @@ source ../common/util.sh
 # Set the number of repetitions in nightly and weekly tests
 # Set the email subject for nightly and weekly tests
 if [ "$TRITON_PERF_WEEKLY" == 1 ]; then
-    REPETITION_CPP=2000000
```

Comment:
Add a comment to clarify how much time these reps correspond to approximately

Reply:
Comments updated.